src/test/scala/nl.knaw.dans.easy.bag2deposit/ddm/RewriteSpec.scala

```diff
@@ -692,13 +693,43 @@ class RewriteSpec extends AnyFlatSpec with XmlSupport with SchemaSupport with Ma
   it should "replace funder role" in {
     val ddmIn = XML.loadFile("src/test/resources/funder/ddm-in.xml")
     val expectedDDM = XML.loadFile("src/test/resources/funder/ddm-out.xml")
+    val triedDdm = new DdmTransformer(File("src/main/assembly/dist/cfg"), Map.empty)
+      .transform(ddmIn, "easy-dataset:123")
+    triedDdm.map(normalized) shouldBe Success(normalized(expectedDDM))
+    assume(schemaIsAvailable)
     validate(ddmIn) should matchPattern {
       case Failure(e) if e.getMessage.contains("Funder") =>
     }
     validate(expectedDDM) shouldBe a[Success[_]]
-    val triedDdm = new DdmTransformer(File("src/main/assembly/dist/cfg"), Map.empty)
-      .transform(ddmIn, "easy-dataset:123")
-    triedDdm.map(normalized) shouldBe Success(normalized(expectedDDM))
+  }
+  it should "accept <..><..><..>" in {
+    (testDir / "ddm-encoding.xml").writeText(
+      printer.format(Utility.trim(
+        ddm(title = "Title <E2><80><93> of the <E2><80><98>dataset<e2><80><99><cf><be>", audience = "D37000", dcmi = <ddm:dcmiMetadata/>)
+      )).replaceAll("&lt;", "<").replaceAll("&gt;", ">")
+    )
+    val triedDdmIn = loadXml(testDir / "ddm-encoding.xml")
+    triedDdmIn shouldBe a[Success[_]]
+
+    val triedDdmOut = new DdmTransformer(File("src/main/assembly/dist/cfg"), Map.empty)
+      .transform(triedDdmIn.get, "easy-dataset:123")
+    triedDdmOut shouldBe a[Success[_]]
+    (triedDdmOut.get \\ "title").text shouldBe "Title – of the ‘dataset’Ͼ"
```

Comment:
Intellij is happy, with mvn I get 

    "Title [??? of the ???dataset?????]" was not equal to "Title [? of the ?dataset??]" (RewriteSpec.scala:717)

Possibly some UTF-8 setting?